raster/r.neighbors/readweights.c

```diff
@@ -42,3 +42,20 @@ void gaussian_weights(double sigma)
 	}
     }
 }
+
+void exponential_weights(double factor)
```

Comment:
Not sure if this is really important, but creating a new function for each algorithm seems to create a lot of redundant code. Why not create one function that takes the choice of weighting algorithm and the "factor" ?